test/manager/gitlabci-include/extract.spec.ts

```diff
@@ -9,21 +8,27 @@ const yamlFile = fs.readFileSync(
 
 describe('lib/manager/gitlabci-include/extract', () => {
   describe('extractPackageFile()', () => {
-    let config: ExtractConfig;
```

Comment:
Note: I removed the config from the existing test cases because it wasn't necessary for them. The config-endpoint mapping is now handled by the new test case